src/components/common/AppBreadcrumbs.tsx

```diff
@@ -102,20 +123,64 @@ const AppBreadcrumbs = (props: Props) => {
     Boolean,
   );
   return (
-    <Breadcrumbs className={classes.root} separator={<NavigateNextIcon fontSize="small" />} aria-label="breadcrumb">
-      {crumbs.map((crumb, i) => (
-        <Crumb
-          key={crumb.name}
-          active={crumbs.length - 1 === i}
-          name={crumb.name}
-          href={crumb.href}
-          Icon={crumb.Icon}
-        />
-      ))}
-    </Breadcrumbs>
+    <Stack className={classes.root} direction="row" spacing={1}>
+      {viewer.relatedOwners && viewer.relatedOwners.length === 1 ? null : <AccountsButton viewer={viewer} />}
+      <Breadcrumbs
+        className={classes.breadcrumbs}
+        separator={<NavigateNextIcon fontSize="small" />}
+        aria-label="breadcrumb"
+      >
+        {crumbs.map((crumb, i) => (
+          <Crumb
+            key={crumb.name}
+            active={crumbs.length - 1 === i}
+            name={crumb.name}
+            href={crumb.href}
+            Icon={crumb.Icon}
+          />
+        ))}
+      </Breadcrumbs>
+    </Stack>
   );
 };
 
+interface AccountsButtonProps extends WithStyles<typeof styles> {
+  viewer: AppBreadcrumbs_viewer;
+}
+
+const AccountsButton = styled(({ viewer, classes }: AccountsButtonProps) => {
+  const navigate = useNavigate();
+
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+  const open = Boolean(anchorEl);
+
+  const handleClick = (e: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(e.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const handleMenuItemClick = (e, name) => {
+    setAnchorEl(null);
+    navigateHelper(navigate, e, '/github/' + name);
+  };
+
+  return (
+    <>
+      <Button className={classes.accountsButton} variant="contained" onClick={handleClick}>
+        Accounts
+        <ArrowDropDownIcon className={classes.accountsButtonIcon} />
```

Comment:
It seems it should be `endIcon` of the `Button`. Probably then you won't need `classes.accountsButtonIcon`.